components/organisms/BiddingTable.tsx

```diff
@@ -23,43 +26,92 @@ const columns = [
   'Highest Bid',
   'My Bid',
   'Status',
-  '', // reserve last column for any button actions
+  'Quick Bid', // reserve last column for any button actions
 ]
 
 export default function BiddingTable({ data }: Props): JSX.Element {
   console.log('data: ', data) // use data variable so eslint doesnt complain
   return (
-    <Table variant="brandDashboard">
-      <Thead>
-        <Tr>
-          {columns.map(col => (
-            <Th key={col}>{col}</Th>
-          ))}
-        </Tr>
-      </Thead>
-      <Tbody>
-        <Tr>
-          <Td>
-            <Flex align="center">
-              <Avatar name="Linus Tech Tips" src="https://bit.ly/dan-abramov" />
-              <Flex direction="column" ml={CREATOR_AVATAR_TEXT_SPACING}>
-                <Box textStyle="body2">Linus Tech Tips</Box>
-                <Box textStyle="caption2">10k viewers</Box>
+    <LinkBox>
+      <Table variant="brandDashboard">
+        <Thead>
+          <Tr>
+            {columns.map(col => (
+              <Th key={col}>{col}</Th>
+            ))}
+          </Tr>
+        </Thead>
+        <Tbody>
+          <Tr>
+            <Td>
+              <LinkOverlay href="./offer-details">
```

Comment:
1. the idea should be that the entire row is clickable (<Tr> in this case). 
2. There shouldn't be a `.` in href